x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -98,7 +98,9 @@
 
         ANALYTICS(OperationMode.MISSING, true),
 
-        SEARCHABLE_SNAPSHOTS(OperationMode.ENTERPRISE, true);
+        SEARCHABLE_SNAPSHOTS(OperationMode.ENTERPRISE, true),
+
+        OPERATOR_PRIVILEGES(OperationMode.ENTERPRISE, true);
```

Comment:
Other than adding the new enum here, what else is needed for telemetry? The `XPackUsageFeatureAction` is for feature usage which means billing and is not telemetry?

Reply:
Thanks. The names are pretty confusing. It's now added. 
Fun fact, the new telemetry action is the first one that trips the test of "every acton is either operator-only or non-operator".